src/Components/Blazor/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Override version labels -->
+    <VersionPrefix>3.2.0</VersionPrefix>
```

Comment:
@javiercn, do we want to do anything clever here to compute the version prefix using these values: https://github.com/aspnet/AspNetCore/blob/blazor-wasm/eng/Versions.props#L9-L11?